src/IntercomContacts.php

```diff
@@ -78,6 +84,38 @@ public function deleteContact(string $id, array $options = [])
         return $this->client->delete($path, $options);
     }
 
+    /**
+     * Applys a tag to a Contact based on the provided Tag ID
+     *
+     * @see    https://developers.intercom.com/intercom-api-reference/reference#tag-contact
+     * @param  string $id
+     * @param  string  $tag_id
+     * @return stdClass
+     * @throws Exception
+     */
+    public function addTag(string $id, string $tag_id)
+    {
+        $path = $this->contactPath($id);
```

Comment:
I think it would be cleaner if we create a `contactTagsPath` method in this class, what do you think?

Reply:
Good call, adding now.